mail_gateway/models/mail_gateway.py

```diff
@@ -25,12 +25,10 @@ class MailGateway(models.Model):
     )
     webhook_user_id = fields.Many2one(
         "res.users",
-        default=lambda self: self.env.user.id,
+        default=lambda self: self.env.ref("base.user_root"),
         help="User that will create the messages",
     )
-    member_ids = fields.Many2many(
-        "res.users", default=lambda self: [Command.link(self.env.user.id)]
```

Comment:
I think this changes are the reason behind the test issue

Reply:
> I think this changes are the reason behind the test issue

The CI has been failing for several days on all PRs/commits in V16.

The last commit is unrelated to WhatsApp templates. If you prefer, I can create a new PR for it separately. However, this commit is not linked to the tests but addresses a different case, please refer to the commit description for more details.

Let me know what you think.